### PR TITLE
[#1781] limiting description and title width in Administration

### PR DESCRIPTION
**Pull request by gbonaspetti**
https://github.com/tracim/tracim/issues/1781

### CODE REVIEW

File: frontend_app_admin_workspace_user/src/css/index.styl [hunk 1]
```diff
@@ -19,9 +19,18 @@
...
     &__td-link
+      max-width 120px
+      white-space nowrap
+      overflow hidden
+      text-overflow ellipsis
       cursor pointer
       &:not(:hover)
         color inherit
+    &__td-description
+      max-width 440px
```
Comment (Skylsmoi) [inline frontend_app_admin_workspace_user/src/css/index.styl]: This value doesn't work on small devices.
You should add some breakpoint to reduce it.

File: frontend_app_admin_workspace_user/src/css/index.styl [hunk 2]
```diff
@@ -19,9 +19,18 @@
     &:hover, &:nth-child(even):hover
       background-color grey-hover
     &__td-link
+      max-width 120px
```
Comment (Skylsmoi) [inline frontend_app_admin_workspace_user/src/css/index.styl]: This value doesn't work on small devices.
You should add some breakpoint to reduce it.

**Comment by lebouquetin** [comment]
Note for both of you: we do not expect admin screen to be responsive. It's acceptable to have admin screens only with a computer-targetted rendering

**Comment by gbonaspetti** [comment]
For this solution to work, we need to solve the issue https://github.com/tracim/tracim/issues/1757 first.

**Pull request merged**